pkg/asset/manifests/aws/cluster.go

```diff
@@ -190,6 +190,13 @@ func GenerateClusterAssets(ic *installconfig.InstallConfig, clusterID *installco
 						ToPort:      22623,
 						CidrBlocks:  []string{capiutils.CIDRFromInstallConfig(ic).String()},
 					},
+					{
```

Comment:
Need to confirm if CAPA is merging this rule correctly, it seems not to be applyed in AWS Console, even though the deployment is finished correctly and wont impact publish=External clusters as there is 0/0 inbound rule to 6443 which will overlap this rule.